doc/todo/RELEASE

```diff
@@ -9,6 +7,89 @@ Following shell expansions are used in this document:
  ~elektra-ftp .. where ftp files go to
  ~elektra-doc .. where doc flies go to
 
+----------------------------------------------------------------------
+
+This describes what to do for the automated release:
+
+= Tasks before Release =
```

Comment:
I think that for this first part it is actually best if we make an GitHub issue for every release in which we tick the things to be done (one by one). This makes it clear, e.g. if the version numbers are already incremented or not.

I created  #3553